nodes/list/get_list_element_index.py

```diff
@@ -0,0 +1,66 @@
+import bpy
+from bpy.props import *
+from ... tree_info import keepNodeLinks
+from ... events import executionCodeChanged
+from ... base_types.node import AnimationNode
+from ... sockets.info import getBaseDataTypeItems, toIdName, toListIdName, isBase, toBaseDataType
+
+class GetListElementIndexNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GetListElementIndexNode"
+    bl_label = "Get Element Index"    #Search List Element ?
```

Comment:
Lets use "Search List Element" :)
Also don't forget to update the bl_label, bl_idname, class name und file name.